solr/core/src/java/org/apache/solr/update/CommitTracker.java

```diff
@@ -57,8 +58,10 @@
   private int docsUpperBound;
   private long timeUpperBound;
   private long tLogFileSizeUpperBound;
-  
-  private final ScheduledExecutorService scheduler = 
+
+  // note: can't use ExecutorsUtil because it doesn't have a *scheduled* ExecutorService.
+  //  Not a big deal but it means we must take care of MDC logging here.
```

Comment:
Is there an issue for improving ExecutorUtil for this?

Reply:
I thought of that.  There isn't much need for a ScheduledExecutorService throughout Solr, and it'd probably add a bunch of code over in ExecutorsUtil to add one.  I looked at all that ExecutorsUtil does which is kind of a ton and thought all I really want here is a basic MDC to set the core for logging and that's it.  That said, if you think the usage here ought to be doing more than just this, then we should probably go add stuff to ExecutorsUtil for the Scheduled case.

Reply:
I think it's fine.